public/app/features/alerting/unified/components/receivers/ReceiversTable.tsx

```diff
@@ -369,17 +375,20 @@ function useGetColumns(
     {
       id: 'name',
       label: 'Contact point name',
-      renderCell: ({ data: { name, provisioned } }) => (
-        <>
-          {name} {provisioned && <ProvisioningBadge />}
-        </>
+      renderCell: ({ data: { name, provisioned, grafanaAppReceiverType } }) => (
```

Comment:
seems you don't need `grafanaAppReceiverType` param here

Reply:
truee